api/src/main/java/org/openmrs/util/DoubleRange.java

```diff
@@ -77,6 +77,39 @@ public void setLow(Double low) {
 		this.low = low == null ? new Double(Double.NEGATIVE_INFINITY) : low;
 	}
 	
+	/**
+	 * @return Returns the closedLow 
+	 * 
+	 */
+	public boolean getclosedLow () {
+		return closedLow;
+	}
+	
+	/**
+	 * @param closedLow to set
+	 * 
+	 */
+	public void closedLow(boolean closedLow) {
+		this.closedLow = closedLow == true;
```

Comment:
simply having `this.closedLow = closedLow` would be enough

Reply:
That's ok i can change this